packages/gatsby-kontent-components/README.md

```diff
@@ -11,9 +11,28 @@ The package containing React components useful when processing Kontent data to t
 ## Install
 
 ```sh
-npm install @kentico/gatsby-kontent-components
+npm install @kentico/gatsby-kontent-components gatsby-plugin-image
 ```
 
+Also, add `gatsby-plugin-image` to `plugins` array in `gatsby-config.js`.
+
+## <a name="image-element-component">Image element component</a>
+
+Images from Kentico Kontent can be displayed using the `ImageElement` component. This wraps the `GatsbyImage` component from [gatsby-plugin-image](https://www.gatsbyjs.com/docs/how-to/images-and-media/using-gatsby-plugin-image/), so ensure that you also install that plugin. This component will give the best experience for your users, as it includes responsive srcset, blur-up, lazy loading and many other performance optimizations. [Automatic format optimization](https://docs.kontent.ai/reference/image-transformation#a-automatic-format-selection) is always enabled. In many cases it can improve Lighthouse scores by 10-20 points.
+
+The component takes all [the `GatsbyImage` props](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image#gatsbyimage), as well as the following properties. All are optional except `image`:
+
+- `image`: the `image` object. This should include `url`, `width` and `height`.
+- `layout`: see [the `gatsby-plugin-image` docs](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image#layout)
+- `width`/`height`: see [the `gatsby-plugin-image` docs](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image#widthheight)
+- `aspectRatio`: see [the `gatsby-plugin-image` docs](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image#aspectratio)
+- `backgroundColor`: displayed as a placeholder while the image loads
+- `options`: an object containing options passed to [the Kontent Image Transformation API](https://docs.kontent.ai/reference/image-transformation). Supported options: `fit`, `quality`, `lossless`.
+
+Properties of the image object (e.g. `width` and `height`) are reflected in Kontent's image API query.
+Props of the `ImageElement` component (e.g. `width` and `height`) are reflected in the rendered DOM.
+If the optional props of `ImageElement` are omitted, the properties of the image object are applied.
+
```

Comment:
I would personally like to see the simple example/snippet of how to use Gatsby image, something like this + link the sample in the dev sample:


```jsx
import React from 'react'
import { ImageElement } from "@kentico/gatsby-kontent-components"
import { graphql } from "gatsby";

export default Page = ({ data }) => {
  const avatar = data.author.elements.avatar_image.value[0];

  return (
    <ImageElement
      image={avatar}
      width={800}
      height={200}
      backgroundColor="#bbbbbb"
      alt={avatar.description}
    />
  );
}
export const query = graphql`
  {
    author: kontentItemAuthor {
      elements {
        avatar_image {
          value {
            url
            description
          }
        }
      }
    }
  }
`
```

Reply:
good point, fixed in https://github.com/Kentico/kontent-gatsby-packages/pull/174/commits/7a3b74beaeef98abe14b40157238834d79ad14e6